src/compounds/legacy-product-table/src/index.tsx

```diff
@@ -9,6 +9,128 @@ import CirclePercentageBar from '@uswitch/trustyle.circular-percentage-bar'
 
 import RowWrapper from './row-wrapper'
 
+interface WireFrameCellProps extends React.HTMLAttributes<HTMLDivElement> {
+  styles?: any
+}
+
+const WireFrameCell: React.FC<WireFrameCellProps> = ({ styles }) => {
+  const defaultStyles = {
+    backgroundColor: 'grey-20',
+    display: 'block'
+  }
+
+  return <span sx={{ ...defaultStyles, ...styles }}></span>
+}
+
+interface WireFrameProps extends React.HTMLAttributes<HTMLDivElement> {
+  type: 'header' | 'additionalInfo' | 'representativeExample'
+}
+
+const WireFrame: React.FC<WireFrameProps> = ({ type }) => {
+  const containerStyles: {
+    header: object
+    additionalInfo: object
+    representativeExample: object
+  } = {
+    header: {
+      paddingLeft: ['15px', '160px'],
```

Comment:
these numbers feel a bit magic, where do they come from, can we get anything from the theme?

Reply:
Turned them into percentages according to the design

Reply:
I got this from the current styling of the header on legacy product table